doc/modules/outlier_detection.rst

```diff
@@ -175,8 +176,54 @@ This strategy is illustrated below.
            Data Mining, 2008. ICDM'08. Eighth IEEE International Conference on.
 
 
-One-class SVM versus Elliptic Envelope versus Isolation Forest
---------------------------------------------------------------
+Local Outlier Factor
+--------------------
+
+One other very efficient way of performing outlier detection in datasets whose
```

Comment:
Maybe `Another efficient way to perform outlier detection on moderately high dimensional datasets is ...` ?